docs/config-files/config.py

```diff
@@ -0,0 +1,14 @@
+#This file contains all of the user based inputs
+import numpy
+
+Circuit_Type = 'RRQRQ' #e.g. 'RC'
```

Comment:
I would generally avoid mixing upper and lower case in variable names.  Do we have control over that?  Often followed standards are for global variables, such as these, to be set to upper case.

Reply:
Sorry about the confusion. I mistakenly upload the config example that Austin gave me previously. The correct one was the following: https://github.com/sunhwibang/pytentiostat/commit/064b7f3f4f8d859a8648245a9562c43d5f23cb64

However, your comments provide lots of good feedback, such as upper/lower case of variable names.

From the last week's meeting, I am working on the config file using a yaml file.